.github/actions/mount-share/action.yml

```diff
@@ -0,0 +1,74 @@
+name: mount-share
+description: Mount Azure file share to linux runner
+inputs:
+  storage-account-resource-group:
+    required: true
+  storage-account-name:
+    required: true
+  file-share-name:
+    require: true
+outputs:
+  mount-path:
+    value: ${{ steps.mount_share.outputs.mount_path }}
+
+runs:
+  using: "composite"
+  steps:
+
+    # Source:
+    # https://docs.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-linux?tabs=smb311#static-mount-with-etcfstab
+    - name: Mount file share
+      id: mount_share
+      shell: bash
+      run: |
+        mntRoot="/mount"
+        sudo mkdir -p $mntRoot
+
+        fileShareName="${{ inputs.file-share-name }}"
+
+        resourceGroupName="${{ inputs.storage-account-resource-group }}"
+        storageAccountName="${{ inputs.storage-account-name }}"
+
+        # Create a folder to store the credentials for this storage account and
+        # any other that you might set up.
+        credentialRoot="/etc/smbcredentials"
+        sudo mkdir -p "/etc/smbcredentials"
```

Comment:
maybe `sudo mkdir -p $credentialRoot` since you already defined the variable?

Reply:
I'm trying to not make any changes to the code [Microsoft provided](https://docs.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-linux?tabs=smb311#automatically-mount-file-shares), just for easier comparison in the future